Overview.src.html

```diff
@@ -381,9 +381,9 @@ <h4 id=terminology-headers>Headers</h4>
  <li>`<code title>Set-Cookie2</code>`
 </ul>
 
-<p>A <dfn>CORS-safelisted response-header name</dfn>, given a
-<span title=concept-response-header-list>header list</span> <var>list</var>, is a
-<span title=concept-header>header</span> <span title=concept-header-name>name</span> that is one of:
+<p>A <dfn id="cors-safelisted-response-header-name">CORS-safelisted response-header name</dfn>, given a
+<span title=concept-response-cors-exposed-headers-list>CORS-exposed headers list</span> <var>list</var>, is a
+<a href="#concept-header" title="concept-header">header</a> <a href="#concept-header-name" title="concept-header-name">name</a> that is one of:
```

Comment:
Yeah, my mistake. Also happy to leave out the Overview.html changes if that's what you prefer, although that wouldn't have prevented my from stupidly managing to copy stuff from the generated html into the source file in an attempt to undo earlier changes I made...